### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Non Text Contrast: Color contrast of the unrated star to its background is having the contrast ratio of 2.1:1 

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2683]
### GitHubTags:
#Azure Tools for VS Code;#A11ySev1;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG1.4.11;#MAC; #NonText Contrast;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Note: 
Similar Issue is observed with Rated star where color contrast of the rate star to its background is having the contrast ratio of 2.4:1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till "rating" link and check whether color contrast of the unrated star to its background is meeting the contrast ratio of >=4.5:1 or not.

### Actual Result:
Color contrast of the unrated star to its background is having the contrast ratio of 2.1:1 

### Expected Result:
Color contrast of the unrated star to its background should be having the contrast ratio of >=4.5:1

### User Impact:
Low Vision users will not be able to know about rating information if Color contrast of the unrated star is having the contrast ratio of 2.1:1 

### Attachment for Reference:
<img width="820" alt="Bug_2683_Color contrast of the rated star is having the contrast ratio of 2 4:1" src="https://user-images.githubusercontent.com/50438094/108317706-af274280-71e4-11eb-9a42-6b126425fb32.png">
<img width="820" alt="Bug_2683_Color contrast ratio of unrated star to its background is having the contrast ratio of 2 1:1" src="https://user-images.githubusercontent.com/50438094/108317746-bb130480-71e4-11eb-938c-8ffd9d1ff71b.png">

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
